spec/correios_sigep/configuration_spec.rb

```diff
@@ -3,82 +3,95 @@ module CorreiosSigep
     subject { described_class.new }
 
     describe '#administrative_code' do
-      it 'return nil when unset' do
-        expect(subject.administrative_code).to_not be
```

Comment:
@downgba can u please change all `to_not` for `not_to`? :P 
cc @fpgentil 
thanks =D